From 057c1c33987528994eb913aafd5f3f7de19eb87d Mon Sep 17 00:00:00 2001
From: David Holsgrove <david.holsgrove@petalogix.com>
Date: Thu, 26 Jan 2012 08:42:46 +1000
Subject: [PATCH 08/41] microblaze fcntl.h: Add definition of F_GETPIPE_SZ

prevents error in building eglibc;

../sysdeps/unix/sysv/linux/fpathconf.c:55:24: error: 'F_GETPIPE_SZ' undeclared (first use in this function)

Signed-off-by: David Holsgrove <david.holsgrove@petalogix.com>
---
 sysdeps/unix/sysv/linux/microblaze/bits/fcntl.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/sysdeps/unix/sysv/linux/microblaze/bits/fcntl.h b/sysdeps/unix/sysv/linux/microblaze/bits/fcntl.h
index e4b6187..071d01e 100644
--- a/sysdeps/unix/sysv/linux/microblaze/bits/fcntl.h
+++ b/sysdeps/unix/sysv/linux/microblaze/bits/fcntl.h
@@ -102,6 +102,8 @@
 # define F_NOTIFY	1026	/* Request notfications on a directory.	 */
 # define F_DUPFD_CLOEXEC 1030	/* Duplicate file descriptor with
 				   close-on-exit set.  */
+# define F_SETPIPE_SZ	1031	/* Set pipe page size array.  */
+# define F_GETPIPE_SZ	1032	/* Set pipe page size array.  */
 #endif
 
 /* For F_[GET|SET]FD.  */
-- 
1.7.1

